src/components/SearchBar/SearchBar.component.jsx

```diff
@@ -0,0 +1,60 @@
+import React, { useState } from 'react';
+
+import { useHistory } from 'react-router-dom';
+// import styled from 'styled-components';
+
+import './SearchBar.styles.css';
+
+export default function SearchBar({ initSearchQuery, onChange, homepage }) {
```

Comment:
We could rename `homepage` to `isHomepage` so it is clearer that the variable holds a boolean value.

Reply:
Done.